docs/developer/extending/webhooks/synchronous-events/stored-payment-method.mdx

```diff
@@ -331,6 +331,14 @@ The next example illustrates the predefined payload that will be used if a subsc
 }
 ```
 
+- A response indicating a pending tokenization:
+
+```json
+{
+  "result": "PENDING"
```

Comment:
Shouldn't we have some `data` here optionally as well? 🤔

Reply:
👮‍♂️ 🚔